.github/workflows/typecheck.yml

```diff
@@ -0,0 +1,27 @@
+on:
```

Comment:
can you please use the aegir template https://github.com/ipfs/aegir/wiki/Github-Actions-Setup

Reply:
Probably remove travis ci also

Reply:
Can we please do the travis to github migration as a separate pr ?